recipes/libigl/2.x.x/conanfile.py

```diff
@@ -52,7 +52,7 @@ def validate(self):
                     self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
         if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime and not self.options.header_only:
             raise ConanInvalidConfiguration("Visual Studio build with MT runtime is not supported")
-        if "arm" in self.settings.arch or "x86" is self.settings.arch:
+        if "arm" in self.settings.arch or "x86" == self.settings.arch:
             raise ConanInvalidConfiguration("Not available for arm. Requested arch: {}".format(self.settings.arch))
```

Comment:
the message doesn't match the condition 🤔 don't need to change in this PR, it was always like that...